docs/01-app/01-getting-started/08-updating-data.mdx

```diff
@@ -172,6 +172,8 @@ There are two main ways you can invoke a Server Function:
 1. [Forms](#forms) in Server and Client Components
 2. [Event Handlers](#event-handlers) and [useEffect](#useeffect) in Client Components
 
+> **Good to know:** Server Functions are designed for server-side mutations. The client currently dispatches and awaits them one at a time. This is an implementation detail and may change. If you need parallel data fetching, use data fetching in Server Components, or perform parallel work inside a single Server Function or Route Handler.
```

Comment:
So this is a weird addition, in the sense that, the docs don't ever recommend Server Functions/Actions for data fetching, but people use them for that, and end up discovering that they were run sequentially...

Reply:
I feel like we really need to link out from this message to somewhere that explains how to do each of these recommendations

Reply:
True, we do have docs for that - I'll update it shortly